tests/storage/migration.go

```diff
@@ -363,6 +363,7 @@ var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSched
 
 		It("should migrate the source volume from a block source and filesystem destination DVs", func() {
 			volName := "disk0"
+			dstSize := "1.2Gi"
```

Comment:
My concern is that this is guaranteed to be a scenario users hit, which we may want to try to accomodate.
Is the volume migration aware of filesystem overhead? maybe there's something we could do on that side? at least for ceph CSI?

I mean we get this PR in and it would handle the test failure ceph rbd block-ceph rbd fs but I am just concerned this is a bigger issue

Reply:
The users could hit this scenario, but then the migration will fail and report the error describe above. They are, de facto, trying to migrate from a larger image to a smaller one.

No, volume migration isn't aware about filesystem overhaead. How would you handle it for ceph CSI?

I don't think it is an issue we can solve. There could be mistmatches between storage classes, so the migration isn't possible if the user doesn't create a larger destination disk as it happens in this case.

IMO, taking into account the filesystem overhead is the goal of the overlying tools who they are actually also creating the target pvc/dv.

Reply:
@akalenyu no it is the test which creates the DataVolume, no the volume migration. 
TBH, the test is dump, it just tries to copy from block to fs. I don't want to do any math to try to guess the correct size. This isn't the goal of that test.

Reply:
I'm trying to help see if we can avoid the math.. if the target FS DV is created with 1Gi it should get rounded up (storage API adds fs overhead on top) and we should not see the failure. I am not sure why this is not what occurs

Reply:
I think I see what happens. This test is using the PVC API and thus the volume is not enlarged according to overhead. WDYT about switching this test to use the storage API ([libdv.WithStorage())](https://github.com/kubevirt/kubevirt/pull/12072)?
```
"spec": {
                "source": {
                    "blank": {}
                },
                "pvc": {
                    "accessModes": [
                        "ReadWriteOnce"
                    ],
```

Reply:
https://github.com/kubevirt/kubevirt/pull/13147

Reply:
@akalenyu not quite sure I understand the comment. It is already using the WithStorage function, see [here](https://github.com/kubevirt/kubevirt/blob/main/tests/storage/migration.go#L383)

Reply:
yep, it does - but not in the release-1.3 branch. so I don't think a fix is needed in main

Reply:
ah I see. Do you want to open an new PR for the `release-1.3` branch,  then I can approve it and we can close this.

Reply:
this is what I did haha ^

Reply:
Monday morning, sorry!